blog.py

```diff
@@ -189,6 +189,7 @@ def _create_vtable_posts(self):
             content,
             empty,
             updated,
+            retitle,
```

Comment:
retitle is a terrible name, you can use `name_title_mismatch`

Reply:
changed to name_title_mismatch